sound/soc/intel/boards/bdw-rt5677.c

```diff
@@ -295,6 +295,14 @@ SND_SOC_DAILINK_DEF(platform,
 SND_SOC_DAILINK_DEF(be,
 	DAILINK_COMP_ARRAY(COMP_CODEC("i2c-RT5677CE:00", "rt5677-aif1")));
 
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_BROADWELL)
+SND_SOC_DAILINK_DEF(ssp0_port,
+	    DAILINK_COMP_ARRAY(COMP_CPU("ssp0-port")));
+#else
+SND_SOC_DAILINK_DEF(ssp0_port,
+	    DAILINK_COMP_ARRAY(COMP_DUMMY())));
```

Comment:
One too many brackets there. Fails to compile

Reply:
@xiulipan @fredoh9 I thought we compiled in SST mode? We have a kconfig-sst.sh, isn't this used?

Reply:
Travis did catch the error
https://travis-ci.org/thesofproject/linux/jobs/633817565#L2610